training/src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -1,13 +1,15 @@
 package wolox.training.repositories;
 
-import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
 import wolox.training.models.Book;
 
-interface BookRepository {
+@Repository
+public interface BookRepository extends CrudRepository<Book,String> {
 
-    @Query("SELECT book FROM Book book WHERE book.author = :name")
+    @Query("SELECT book FROM Book book WHERE book.author = :author")
     Book findBookByAuthor(@Param("author") String author);
```

Comment:
What happens if there are several books with the same author? Does it pick one or does it break?